Sources/ArgumentParser/Usage/UsageGenerator.swift

```diff
@@ -301,8 +301,21 @@ extension ErrorMessageGenerator {
     }
   }
   
-  func duplicateExclusiveValues(previous: InputOrigin, duplicate: InputOrigin) -> String? {
-    return "Value at position \(duplicate) has already been set from value at position \(previous)."
+  func duplicateExclusiveValues(previous: InputOrigin, duplicate: InputOrigin, arguments: [String]) -> String? {
+    func elementString(_ origin: InputOrigin, _ arguments: [String]) -> String? {
+      guard case .argumentIndex(let split) = origin.elements.first else { return nil }
+      var argument = "\"\(arguments[split.inputIndex.rawValue])\""
```

Comment:
We're using single quotes around user input in the other messages, can you do the same here?

Reply:
I had failed to notice. Done!